content/lessons/chapter-4/public-key-3.tsx

```diff
@@ -46,41 +59,42 @@ const javascript = {
 const python = {
   program: `print("KILL")`,
   defaultFunction: {
-    name: 'find_hash',
-    args: ['nonce'],
+    name: 'rawPublicKey',
+    args: ['private_key'],
   },
-  defaultCode: `# Import ECDSA library.
-from lib import secp256k1
-  
-def privatekey_to_publickey(private_key):
-    # Multiply the private key by the ECDSA generator point G to
-    # produce a new curve point which is the public key.
-    # Return that curve point (also known as a group element)
-    # which will be an instance of secp256k1.GE
-    # See the library source code for the exact definition
-    G = secp256k1.G
-`,
-  //   defaultCode: `from hashlib import sha256
-
-  // # Create a program that finds a sha256 hash starting with 5 zeroes.
-  // # To submit your answer, print it to the terminal using print().
-
-  // # Type your code here
-  // `,
+  defaultCode: `${secp256k1.secp256k1py}
+# Import ECDSA library.
+# Multiply the private key by the ECDSA generator point G to
+# produce a new curve point which is the public key.
+# Return that curve point (also known as a group element)
+# which will be an instance of secp256k1.GE
+# See the library source code for the exact definition
+# https://github.com/saving-satoshi/challenges/blob/master/chapter4/python/lib/secp256k1.py
+G = SECP256K1.FAST_G
+# To submit your answer, print to the terminal using print()
+
+
+ `,
```

Comment:
I think we should get rid of these in QA I am not a fan of all the extra lines.there should probably only be one extra